lib/maths/unittest/CTimeSeriesDecompositionTest.cc

```diff
@@ -469,88 +469,89 @@ void CTimeSeriesDecompositionTest::testMinimizeLongComponents() {
 void CTimeSeriesDecompositionTest::testWeekend() {
     double weights[] = {0.1, 0.1, 1.0, 1.0, 1.0, 1.0, 1.0};
 
-    TTimeVec times;
-    TDoubleVec trend;
-    for (core_t::TTime time = 0; time < 100 * WEEK; time += HALF_HOUR) {
-        double weight = weights[(time / DAY) % 7];
-        double daily = 100.0 * std::sin(boost::math::double_constants::two_pi *
-                                        static_cast<double>(time) /
-                                        static_cast<double>(DAY));
-        times.push_back(time);
-        trend.push_back(weight * daily);
-    }
+    for (auto offset : {0 * DAY, 5 * DAY}) {
```

Comment:
This is just the same test performed for different offsets of the start time: the bulk of the changes are indentation.